go/adbc/drivermgr/wrapper.go

```diff
@@ -186,6 +190,19 @@ func getRdr(out *C.struct_ArrowArrayStream) (array.RecordReader, error) {
 	return rdr.(array.RecordReader), nil
 }
 
+func getSchema(out *C.struct_ArrowSchema) (*arrow.Schema, error) {
+	// Maybe: ImportCArrowSchema should perform this check?
+	if out.format == nil {
+		return nil, nil
+	}
+
+	schema, err := cdata.ImportCArrowSchema((*cdata.CArrowSchema)(unsafe.Pointer(out)))
+	if err != nil {
+		return nil, err
+	}
+	return schema, nil
```

Comment:
Why not just `return cdata.ImportCArrowSchema(....)` ?

Reply:
Ha... Good point, must have just been going through the motions. Just pushed the proposed change.